.github/workflows/dagger.yml

```
@@ -0,0 +1,42 @@
+        runs-on: arc-shared
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+              with:
+                fetch-depth: 1
+                clean: true
+                sparse-checkout: |
+                    ${{ inputs.module }}/
+                
+            - run:
+                find 
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 18 but found 16

(indentation)